### PR TITLE
2.4: Updated references to settings locations

### LINKED ISSUE #858: 2.4: settings in web UI are now organised into multiple tabs

### PR DESCRIPTION
**Pull request by degville**
closes #858

### CODE REVIEW

File: en/nodes-kernels.md [hunk 1]
```diff
@@ -111,9 +111,9 @@ for how to perform these three configurations from the CLI.
 ### Default minimum kernel
 
 To set the default minimum enlistment and commissioning kernel (based on Ubuntu
-release: GA kernel) for all machines visit the 'Settings' page and select a
-kernel in the 'Default Minimum Kernel Version' field. Don't forget to click
-'Save'.
+release: GA kernel) for all machines visit the 'General' tab of the 'Settings'
+page and select a kernel in the 'Default Minimum Kernel Version' field of the
+*Commissioning* section	. Don't forget to click 'Save'.
```
Comment (pmatulis) [inline en/nodes-kernels.md]: Superfluous whitespace.

File: en/intro-concepts.md [hunk 2]
```diff
@@ -361,7 +361,8 @@ from 'Deployed' (or 'Allocated') to 'Ready'. Includes action 'Power off'.
 
 The user has the opportunity to erase the node's storage (disks) before
 confirming the action. A default erasure setting can be configured on the
-Settings page. See [Disk erasure][storage-erasure] for details.
+Storage tab of the Settings page. See [Disk erasure][storage-erasure] for
```
Comment (pmatulis) [inline en/intro-concepts.md]: Single quotes around Storage and Settings for consistency.

File: en/nodes-kernels.md (hunk 1 above)
Comment (degville) [reply en/nodes-kernels.md]: thanks!

File: en/intro-concepts.md (hunk 2 above)
Comment (degville) [reply en/intro-concepts.md]: yep.

**Pull request merged**